test/integration/apiserver.go

```diff
@@ -0,0 +1,359 @@
+// Copyright 2017 Istio Authors
```

Comment:
It should probably be `test/integration/apiserver_test.go` just to ensure that it doesn't accidentally get pulled into a binary.